test.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Red Squares Grid</title>
+    <link rel="stylesheet" href="test.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Fira+Mono:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="background-layer"></div>
+    <div class="grid-container">
+        <!-- 24 squares (8x3) -->
+        <div class="grid-cross-item"></div>
+        <div></div>
+        <div class="grid-fill-item"></div>
+        <div></div>
+        <div></div>
+        <div></div>
+        <div></div>
+        <div></div>
+
+        <div></div>
+        <div></div>
+        <div></div>
+        <div></div>
+        <div></div>
+        <div></div>
+        <div></div>
+        <div class="grid-cross-item"></div>
+        
+        <div></div>
+        <div></div>
+        <div></div>
+        <div class="grid-cross-item"></div>
+        <div></div>
+        <div class="grid-cross-item"></div>
+    </div>
+
+    <div class="bottom-left">
+        <p class="fira-mono-regular">&lt;!-- UNSW DATA SCIENCE SOCIETY --&gt;</p>
+        <p class="fira-mono-regular">&lt;!-- MAIN PAGE --&gt;</p>
+        <p class="fira-mono-regular">&lt;!-- ABOUT US --&gt;</p>
+        <br>
+        <p class="fira-mono-regular">© UNSW DATASOC 2024</p>
+    </div>
+
+    <div class="center-grid-container">
+        <div>
+            <img src="./datasoc.svg" alt="Data Soc Logo" class="logo">
+        </div>
+        <div>
+            <p class="username">login&gt;<span>_</span></p>
+        </div>
+        <div">
+            <p class="headingStuff">CMS PORTAL</p>
```

Comment:
should be a h1 -- using the right semantic HTML tags help code readability and SEO